### PR TITLE
Add and edit functionality for Tag

### PR DESCRIPTION
**Pull request by rathorevaibhav**
## Summary

This completes the create & edit functionality for tags using ApolloGraphQL client.

## Test Plan

To be done separately.

### CODE REVIEW

File: src/containers/Tag/Tag.tsx [hunk 1]
```diff
@@ -1,58 +1,159 @@
...
-      id: Number(tag ? tagId : Math.floor(Math.random() * Math.floor(100))),
-      label: name,
+    const payload = {
+      label: label,
       description: description,
-      is_active: isActive,
-      is_reserved: isReserved,
-      language_id: Number(languageId),
-      parent_id: Number(parentId),
+      isActive: isActive,
```
Comment (kurund) [inline src/containers/Tag/Tag.tsx:138]: Any specific reason for changing the keys and removing the type definition for this?

Comment (rathorevaibhav) [reply src/containers/Tag/Tag.tsx:138]: Yeah. Couple of reasons:
1. The GraphQL requires camelCased keys in the payload.
2. in the type definition, we had id, which we don't need to pass to the API. It'll be set automatically.

Comment (kurund) [reply src/containers/Tag/Tag.tsx:138]: I think it's a good practice to keep types. We can do the following:
* Update types to reflect these changes.
* Remove id

Comment (rathorevaibhav) [reply src/containers/Tag/Tag.tsx:138]: Makes sense. Working on it.

Comment (rathorevaibhav) [reply src/containers/Tag/Tag.tsx:138]: This is complete. Ready for review again.

**Pull request merged**